packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -176,10 +176,6 @@ export default class DataGrid extends React.Component {
 	}
 
 	getAgGridConfig() {
-		if (this.props.getRowDataFn && this.props.rowData) {
-			console.warn('Should use either rowData nor getRowDataFn. Not the both at the same time');
-		}
-
```

Comment:
This was polluting the console, because it's `always` outputing when `props.rowData` is provided.... because `props.getRowDataFn` has a default value :/